integrations/postcss/next.test.ts

```diff
@@ -0,0 +1,167 @@
+import { expect } from 'vitest'
+import { candidate, css, js, json, test } from '../utils'
+
+test(
+  'production build',
+  {
+    fs: {
+      'package.json': json`
+        {
+          "dependencies": {
+            "react": "^18",
+            "react-dom": "^18",
+            "next": "14.2.5"
```

Comment:
Do we want to pin this version?

Reply:
Not really, we can do a major pin like we do with react :)